client/lib/abtest/active-tests.js

```diff
@@ -77,4 +77,12 @@ module.exports = {
 		},
 		defaultVariation: 'add'
 	},
+	headstart: {
+		datestamp: '20160203',
+		variations: {
+			original: 20,
+			headstart: 80
```

Comment:
If we use it in all Signup tests, we can insure that our control (original) results aren't muddled by future tests further down the road. For instance, if we were testing a project like a new Trampoline, we'd want users that hadn't been in the control or variant of a previous test (positive new Trampoline results could skew the control of the Headstart test), so we'd check for 'notTested' from Headstart (and other tests) before assigning the new Trampoline test.